### DISCUSSION TITLE
v0.21.0 has been released!

### DISCUSSION THREAD

**Discussion by Pat-Lafon** [lalrpop/lalrpop#934] [Announcements]
See the release notes for more details: https://github.com/lalrpop/lalrpop/blob/master/RELEASES.md#0210-2024-05-30.

Note that we are trying out `cargo-release` for creating LALRPOP releases. Please let myself or @dburgener know if there are any issues.

**Comment by dburgener** [comment]
Awesome!  Thanks @Pat-Lafon!

**Comment by dburgener** [comment]
I switched my Cargo.toml in an external project over to the new version, and the 0.21 pulled in fine for both lalrpop and lalrpop-util, so everything looks like it went smoothly with the release.
